pex/resolve/lockfile/updater.py

```diff
@@ -619,6 +619,7 @@ def create(
         max_jobs,  # type: int
         use_pip_config,  # type: bool
         dependency_configuration,  # type: DependencyConfiguration
+        preserve_log,  # type: bool
```

Comment:
When investigating this I needed to examine Pip logs. The `--preserve-pip-download-log` wasn't working for `pex3 lock sync` run 2+ which prompted this otherwise unrelated fix in this file and the `pex/resolve/resolver_options.py` below.